projects/RabbitMQ.Client/client/api/PublicationAddress.cs

```diff
@@ -116,6 +116,22 @@ public static PublicationAddress Parse(string uriLikeString)
             return null;
         }
 
+        public static PublicationAddress TryParse(string uriLikeString) {
```

Comment:
Various `TryParse` methods usually return a `bool` indicating success, and their second argument is an `out` variable for the result.

Reply:
For instance: [link](https://docs.microsoft.com/en-us/dotnet/api/system.datetime.tryparse?view=netcore-3.1#System_DateTime_TryParse_System_String_System_DateTime__)